code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -97,7 +97,6 @@
 	START_PROCESSING(SSfastprocess, src)
 	GLOB.pai_list += src
 	make_laws()
-	canmove = 0
```

Comment:
not updating mobility flags here?

Reply:
the mob is already created there's no need, it should be NONE while not in shell anyways.